### PR TITLE
Add quickstart for Tensorflow

### PR DESCRIPTION
**Pull request by chenmoneygithub**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/chenmoneygithub/mlflow/pull/10398?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10398/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10398
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?

Add quickstart for Tensorflow. 

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [ ] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,1733 @@
...
+   },
+   "execution_count": 14,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Set up tracking/visualization tool\n",
+    "\n",
+    "In this tutorial, we will use Databricks CE as MLflow tracking server. For other options such as using your local MLflow server, please read the [Tracking Server Overview]().\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb:1303]: Missing the link here

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 2]
```diff
@@ -0,0 +1,1733 @@
...
+    "\n"
+   ],
+   "metadata": {
+    "id": "MS53qvIy_R8g"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "After successfully registering an account on Databricks CE, let's connnect MLflow to Databricks CE. You will need to enter following information at prompt:\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb:1316]:
```suggestion
    "After successfully registering an account on Databricks CE, let's connnect MLflow to Databricks CE. You will need to enter following information at the prompt:\n",
```

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 3]
```diff
@@ -0,0 +1,1733 @@
...
+     "metadata": {},
+     "execution_count": 15
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "While your training is ongoing, you can find this training in your dashboard. Log in to your [Databricks CE](https://community.cloud.databricks.com/) account, and click on top left to select machine learning in the drop down list. Then click on the experiment icon. See the screenshot below:\n",
+    "![landing page](https://drive.google.com/uc?export=view&id=1QxVaolr-L-w96pKUOiYQut3aSRE-04tC)\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb:1520]: We should embed the images within the webserver. I think the traffic volume might flag this drive account, right?

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 4]
```diff
@@ -0,0 +1,1733 @@
...
+    "\n",
+    "After clicking the `Experiment` button, it will bring you to the experiment page, where you can find your runs. Clicking on the most recent experiment and run, you can find your metrics there, similar to:\n",
+    "![experiment page](https://drive.google.com/uc?export=view&id=1wrkSsrGWWQc-ykkGrH86lTzP_66HGhID)\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb:1523]: same here

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 5]
```diff
@@ -0,0 +1,1733 @@
...
+   "metadata": {
+    "id": "WaU6fUiE5nJT"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Customize the MLflow Callback\n",
+    "\n",
+    "If you want to add extra logging logic, you can customize the MLflow callback. You can either subclass from `keras.callbacks.Callback` and write everything from scratch or subclass from `mlflow.tensorflow.MLflowCallback` to change the part you as you need.\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb]:
```suggestion
    "If you want to add extra logging logic, you can customize the MLflow callback. You can either subclass from `keras.callbacks.Callback` and write everything from scratch or subclass from `mlflow.tensorflow.MLflowCallback` to change the portions of the callback as you see fit.\n",
```

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 6]
```diff
@@ -0,0 +1,1733 @@
...
+     "text": [
+      "2023/11/15 01:57:50 WARNING mlflow.utils.autologging_utils: Encountered unexpected error during tensorflow autologging: MLflow autologging must be turned off if an `MLflowCallback` is explicitly added to the callback list. You are creating an `MLflowCallback` while having autologging enabled. Please either call `mlflow.tensorflow.autolog(disable=True)` to disable autologging or remove `MLflowCallback` from the callback list. \n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Going to your Databricks CE page, you should find the `log_loss` is replacing the `loss` metric, similar as the screen shot below.\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb]:
```suggestion
    "Going to your Databricks CE page, you should find that the metric `log_loss` is replacing the `loss` metric, similar to what is shown in the screenshot below.\n",
```

File: docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb [hunk 7]
```diff
@@ -0,0 +1,1733 @@
...
+    "All you need to do is to call `mlflow.tensorflow.autolog()` before kicking off the training, then the backend will automatically log the metrics into the server you configured earlier. In our case, Databricks CE."
+   ],
+   "metadata": {
+    "id": "2Jy-qVMNF24h"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Pick up a name you like.\n",
```
Comment (BenWilson2) [inline docs/source/deep-learning/tensorflow/quickstart/get_started_with_mlflow_and_tensorflow.ipynb]:
```suggestion
    "# Choose any name that you like.\n",
```

File: docs/source/deep-learning/tensorflow/index.rst [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+-------------------------------------------------------
+
+To get a quick overview of how to use the MLflow Tensorflow flavor, please read the quickstart guide. It
+will walk you through the basics of tracking Tensorflow experiments.
+
+.. raw:: html
+
+    <a href="quickstart/quickstart_tensorflow.html" class="download-btn">View the Quickstart</a>
+
+To download the transformers tutorial notebooks to run in your environment, click the respective links below:
```
Comment (jessechancy) [inline docs/source/deep-learning/tensorflow/index.rst:28]: I think this should be tensorflow tutorial notebook?

Comment (chenmoneygithub) [reply docs/source/deep-learning/tensorflow/index.rst:28]: ah good catch!

**Review by BenWilson2 — APPROVED** [review]
LGTM!

**Pull request merged**